tools/etcd-dump-metrics/etcd.go

```diff
@@ -53,11 +53,14 @@ func setupEmbedCfg(cfg *embed.Config, curls, purls, ics []url.URL) {
 	cfg.ListenClientUrls, cfg.AdvertiseClientUrls = curls, curls
 	cfg.ListenPeerUrls, cfg.AdvertisePeerUrls = purls, purls
 
-	cfg.InitialCluster = ""
-	for i := range ics {
-		cfg.InitialCluster += fmt.Sprintf(",%d=%s", i, ics[i].String())
+	if len(ics) > 0 {
+		builder := new(strings.Builder)
+		builder.WriteString(fmt.Sprintf("%d=%s", 0, ics[0].String()))
```

Comment:
See my previous comment. And, in this case, it could also be simplified to:

```go
fmt.Sprintf("0=%s", ics[0])
```

Reply:
```go
fmt.Sprintf("default=%s", &purls[0]
```
See my comment above.